src/integrationTest/java/com/mongodb/kafka/connect/MongoSourceConnectorTest.java

```diff
@@ -214,17 +313,22 @@ void testSourceCanHandleNonExistentCollectionAndSurviveDropping() throws Interru
     void testSourceLoadsDataFromCollectionDocumentOnly() {
         MongoCollection<Document> coll = getDatabaseWithPostfix().getCollection("coll");
 
+        List<Document> docs = insertMany(rangeClosed(1, 50), coll);
+
         Properties sourceProperties = new Properties();
         sourceProperties.put(MongoSourceConfig.DATABASE_CONFIG, coll.getNamespace().getDatabaseName());
         sourceProperties.put(MongoSourceConfig.COLLECTION_CONFIG, coll.getNamespace().getCollectionName());
         sourceProperties.put(MongoSourceConfig.PUBLISH_FULL_DOCUMENT_ONLY_CONFIG, "true");
+        sourceProperties.put(MongoSourceConfig.COPY_EXISTING_CONFIG, "true");
         addSourceConnector(sourceProperties);
 
-        List<Document> docs = insertMany(rangeClosed(1, 100), coll);
         assertProduced(docs, coll);
```

Comment:
Here we test all the documents are shaped as expected - this tests the underlying copying aggregation pipeline.